src/cf-layout.less

```diff
@@ -1203,16 +1271,28 @@
         });
     }
 
+    &__flush {
+        margin-top: 0 !important;
+        margin-bottom: 0 !important;
+        margin-right: -(@grid_gutter-width / 2);
+        margin-left: -(@grid_gutter-width / 2);
+        .respond-to-min(600px, {
+          margin-right: -@grid_gutter-width;
+          margin-left: -@grid_gutter-width;
```

Comment:
Fixed in https://github.com/anselmbradford/cf-layout/commit/84dbcf70811ef69aab1bae4a69e49f4030d80c2a